src/translate-adapter-handlers.ts

```diff
@@ -132,6 +136,18 @@ export async function setDirectories(options: {
 			i18nBases = [defaultPath];
 		}
 	}
+
+	if (options.languages?.length) {
```

Comment:
I think it would be better to have this method always set `translateLanguages`, like all directories. This way we don't need the method `getLanguages()` anymore but can simply reference the variable.